src/Test/Unit/Service/EolValidatorTest.php

```diff
@@ -169,6 +179,8 @@ public function testValidateServiceEol()
      */
     public function testValidateNoticeMessage()
     {
+        Carbon::setTestNow(Carbon::create(2019, 11, 1));
```

Comment:
Does this need to be here or could you just change the test date to be later?

Reply:
This is the only test where a different 'test date' is required to produce the desired result. That, or I could remove this line and modify the EOL for the service in the _file/eol_2.yaml file. I figured setting a different EOL for the service in the config file for the purposes of the test may cause confusion.

Reply:
kk, then makes sense to leave as is.